package.json

```diff
@@ -32,16 +32,16 @@
     "postenv:start": "bash ./tests/e2e/config/env-post-setup.sh"
   },
   "devDependencies": {
-    "@playwright/test": "^1.44.1",
+    "@playwright/test": "^1.54.1",
     "@woocommerce/dependency-extraction-webpack-plugin": "^3.0.1",
-    "@woocommerce/e2e-utils-playwright": "^0.2.1",
+    "@woocommerce/e2e-utils-playwright": "^0.4.0",
     "@woocommerce/eslint-plugin": "^2.3.0",
-    "@woocommerce/woocommerce-rest-api": "^1.0.1",
-    "@wordpress/env": "^10.0.0",
+    "@woocommerce/woocommerce-rest-api": "^1.0.2",
+    "@wordpress/env": "^10.26.0",
     "@wordpress/scripts": "^24.6.0",
```

Comment:
Curious if there's any reason we aren't updating this to the latest, which I think may solve some of the overrides we're having to put in place

Reply:
@dkotter I tried it, but there's a React version conflict with `@wordpress/data` and `@woocommerce/product-editor`. The approach I took was to override non-conflicting dependencies to the latest safe version without having to work on resolving conflicts.